manifests/monitoring/alertmanager-rules.yaml.raw

```diff
@@ -204,6 +204,20 @@ spec:
             cluster:node_cpu_seconds_total:rate5m / count(sum(node_cpu_seconds_total)
             BY (instance, cpu))
           record: cluster:node_cpu:ratio
+    - name: log-counts
+      rules:
```

Comment:
I think the recording rule and the alerting rule will need to be split, the recording should only happen every 24h and save into :elastic_documents_by_namespace_cluster_node:1d while the alerting should happen at a regular interval. 

The alert can then do an average over just the 7 samples

Can we also add the current and historic rate into the label?